tier4_planning_msgs/msg/PlanningFactor.msg

```diff
@@ -0,0 +1,24 @@
+# msg format version
+string version_msg_format
+
+# constants for behavior
+uint16 UNKNOWN = 0
+uint16 SLOW_DOWN = 1
+uint16 STOP = 2
+uint16 GO = 3
+uint16 LEFT_SHIFT = 4
+uint16 RIGHT_SHIFT = 5
+uint16 STRAIGHT = 6
+
+# variables
+string module
+
+bool is_driving_forward
```

Comment:
What does this represent? What is different from behavior GO and STRAIGHT?

Reply:
There is a request to check the driving direction from P/C team. He may want to see module decision, not actual ego behavor. So, I added this flag.